awx/main/tasks.py

```diff
@@ -969,6 +969,9 @@ def build_execution_environment_params(self, instance, private_data_dir):
         if settings.IS_K8S:
             return {}
 
+        if not settings.RUNNER_PROCESS_ISOLATION:
+            return {"process_isolation": False}
```

Comment:
I can't imagine this would work right because project updates and other job types use `container_volume_mounts`... and I don't think those are compatible with setting this to False, but I'm not sure what the actual ansible-runner behavior would be in that case.

Reply:
Hm, I believe you're correct that these options are incompatible with each. I'll check that now.

However, I think this can still work if the `container_volume_mounts` option is removed as well when this is false, since the job will be running directly under the parent process and so will have access to the project directory. I'll do some more testing to make sure this is the case though.

Reply:
@AlanCoding I did confirm that this does work as I expect. The one thing to note is that the default value for `AWX_ISOLATION_BASE_PATH` doesn't work when this setting is set to `False` because container's don't share a `/tmp` location. However, by setting `AWX_ISOLATION_BASE_PATH` to a shared volume mounted in both containers this new setting worked exactly as intended.